.github/workflows/terraform_provider.yml

```diff
@@ -70,21 +70,37 @@ jobs:
         go build -v .
 
   # run tfproviderlint
-  linter:
-    name: Run tfproviderlint
-    needs: build
-    runs-on: ubuntu-latest
-    steps:
-
-    - name: Harden Runner
-      uses: step-security/harden-runner@a4aa98b93cab29d9b1101a6143fb8bce00e2eac4 # v2.7.1
-      with:
-        egress-policy: audit
-
-    - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
-    - uses: bflad/tfproviderlint-github-action@3d16e28385eb2783700821cdf1b70f9f318bed3d # master
-      with:
-        args: ./...
+#  linter:
+#    name: Run tfproviderlint
+#    needs: build
+#    runs-on: ubuntu-latest
+#    steps:
+
+#    - name: Harden Runner
+#      uses: step-security/harden-runner@a4aa98b93cab29d9b1101a6143fb8bce00e2eac4 # v2.7.1
+#      with:
+#        egress-policy: audit
+
+#    - name: Check out code into the Go module directory
+#      uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
+
+#    - name: Set up Go
+#      uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
+#      with:
+#        cache: true
+#        go-version-file: 'go.mod'
+#      id: go
+      
+#    - run: |
+#        go version
+
+#    - name: Install Provider Linter
+#      run: |
+#        go install github.com/bflad/tfproviderlint/cmd/tfproviderlint@latest
+
+#    - name: Run tfproviderlint
+#      run: |
+#        tfproviderlint ./...
```

Comment:
is there a reason to disable these steps? if so, I would remove them from the workflow..

Reply:
Thank you for asking, the lint is not working, is failing, we are having this issue related to go 1.22: https://github.com/bflad/tfproviderlint/issues/298

tfproviderlint -R014=false -AT001.ignored-filename-suffixes=_data_source_test.go ./...
  shell: /usr/bin/bash -e {0}
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference.......... 

We agree to disable and see if the issue is solved, and the lint will work.

Do you prefer to remove this lint part, we also have other parts in this WF commented.

Reply:
As we discuse we will keep commented, once we implement a second lint we can remove this part.